KS.Fiks.IO.Client/Amqp/AmqpHandler.cs

```diff
@@ -21,6 +21,7 @@ internal class AmqpHandler : IAmqpHandler
         private readonly KontoConfiguration _kontoConfiguration;
         private readonly SslOption _sslOption;
         private readonly IConnectionFactory _connectionFactory;
+        private readonly IAmqWatcher _mqWatcher;
```

Comment:
Please use amqp as name in all fields, class-names etc. since this is the naming convention in the rest of the project. Missing a p  :)